<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LsDyna to APDL converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>I/O File</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_original">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>400</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Load file to convert...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Convert">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>92</y>
       <width>106</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_converted">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>400</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Converted file...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadFile">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>46</y>
       <width>106</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Load file...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_original">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Original file</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_converted">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>75</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Converted file</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Information original file:</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="Information">
      <item>
       <widget class="QLabel" name="dimensionfile">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="information_file">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Information converted file:</string>
    </property>
    <widget class="QLabel" name="FileInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>219</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="Nodeinfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>219</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Preview">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionInformazioni"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInformazioni">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
